ConvivaTestApp/src/androidTest/java/com/bitmovin/analytics/conviva/testapp/PlaybackControlsTrackingTests.kt

```diff
@@ -82,11 +82,8 @@ class PlaybackControlsTrackingTests: TestBase() {
         activityScenario.onActivity { activity: MainActivity ->
             verifyOrder {
                 videoAnalyticsMock?.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.SEEK_STARTED,120000)
-                // TODO - why is this behaviour different?
-                // videoAnalyticsMock?.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.PLAYER_STATE, ConvivaSdkConstants.PlayerState.BUFFERING)
+                videoAnalyticsMock?.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.PLAYER_STATE, ConvivaSdkConstants.PlayerState.BUFFERING)
```

Comment:
This test should also check for PLAYING state after SEEK_ENDED. Also please add a new test case like `vodSeekInPauseState` to test same scenario when seek is done in paused state.